packages/webapp/src/pages/EnvironmentSettings.tsx

```diff
@@ -372,7 +372,7 @@ export const EnvironmentSettings: React.FC = () => {
     };
 
     const connectSlack = async () => {
-        const connectionId = `account-${accountUUID}`;
```

Comment:
is this retroactive somehow?

Reply:
No. Going to manually update the prod connections. Only 20 or so